socorro/unittest/conftest.py

```diff
@@ -33,4 +40,26 @@ def test_something(metricsmock):
 @pytest.fixture
 def db_conn():
```

Comment:
This now handles creating a connection, but also truncating certain tables before and after a test. crontabber apps use this, but also the `BetaVersionRule` processor rule tests.